sencha-workspace/SlateDemonstrationsStudent/app/controller/Dashboard.js

```diff
@@ -15,15 +15,66 @@ Ext.define('SlateDemonstrationsStudent.controller.Dashboard', {
 
    // controller configuration
     views: [
-        'OverviewWindow'
+        'OverviewWindow',
+        'RecentProgress',
+        'CompetencyCard'
```

Comment:
@aliyeysides  The `CompetencyCard` type doesn't get created from this controller, that type gets used in `SlateDemonstrationsStudent.view.Dashboard` where it's already being required.

There are three view types that get created by this controller: `Dashboard`, `RecentProgress`, and `OverviewWindow` (in that same order as they're defined within this very file)

Currently when you load the app this error appears in the console because the `Dashboard` view isn't being required anywhere and ExtJS has to freeze the app, guess where the view might be, and synchronously load it. Luckily it works in this case, but warnings like these are not to be ignored and added by PRs

<img width="1268" alt="screen shot 2016-06-15 at 9 55 56 am" src="https://cloud.githubusercontent.com/assets/458494/16089433/e62cde6e-32df-11e6-9bfc-8791b55b7cbc.png">